src/content/docs/apis/nerdgraph/examples/nerdgraph-streaming-export.mdx

```diff
@@ -47,70 +47,82 @@ NRQL requirements:
 
 To set up streaming data export to AWS, you must first set up an Amazon Kinesis Firehose. We'll walk you through that procedure in the next three steps.
 
-### Step 1. Create a Firehose for streaming export [#create-firehose]
+<Steps>
+  <Step>
+### Create a Firehose for streaming export [#create-firehose]
 
 Create a dedicated Firehose to stream your New Relic data to:
 
 1. Go to Amazon Kineses Data Firehose.
 2. Create a delivery stream.
 3. Name the stream (you will use this name later in the rule registration).
 4. Use <DNT>**Direct PUT or other sources**</DNT> and specify a destination compatible with New Relic's JSON event format (for example, S3, Redshift, or OpenSearch).
-
-### Step 2. Create IAM Firehose write access policy [#create-policy]
+  </Step>
+  <Step>
+### Create IAM Firehose write access policy [#create-policy]
 
 1. Go to the IAM and select <DNT>**Policies**</DNT>.
 2. Create a policy.
 3. Select the Firehose service, and then select `PutRecord` and `PutRecordBatch`.
 4. For `Resources`, select the delivery stream, add ARN, and select the region of your stream.
 5. Enter your AWS account number, and then enter your desired delivery stream name in the name box.
 6. Create the policy.
-
-### Step 3. Create IAM role for granting New Relic write access [#iam-role]
-
+  </Step>
+  <Step>
+### Create IAM role for granting New Relic write access [#iam-role]
+               
 To set up the IAM role:
-
+      
 1. Navigate to the IAM and click on <DNT>**Roles**</DNT>.
 3. Create a role for an AWS account, and then select <DNT>**for another AWS account**</DNT>.
```

Comment:
Step 2 is missing here.
I'd follow the instructions from here: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user.html

Reply:
Somehow step 2 was removed, but I've restored it. The instructions are pretty similar to the ones you linked to.